WebAPIDemo1/Model/Book.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebAPIDemo1.Model
+{
+    public class Book
+    {
+        public string bookName { get; set; }
```

Comment:
Naming Convention , it should have UpperCamelCase.
instead of bookName Name would be better